crates/saya/core/Cargo.toml

```diff
@@ -16,24 +16,29 @@ saya-provider.workspace = true
 
 anyhow.workspace = true
 async-trait.workspace = true
-convert_case.workspace = true
+cairo-proof-parser.workspace = true
 cairo-vm.workspace = true
+convert_case.workspace = true
 flate2.workspace = true
 futures.workspace = true
-lazy_static = "1.4.0"
+lazy_static.workspace = true
 parking_lot.workspace = true
 rand = { version = "0.8.5", features = [ "small_rng" ] }
 serde.workspace = true
 serde_json.workspace = true
 serde_with.workspace = true
+starknet-types-core = { version = "0.0.9", default-features = false, features = [ "curve", "num-traits", "serde" ] }
 starknet.workspace = true
 starknet_api.workspace = true
 thiserror.workspace = true
 tokio.workspace = true
 tracing.workspace = true
 url.workspace = true
-starknet-types-core = { version = "0.0.9", default-features = false, features = ["serde", "curve", "num-traits"] }
 
 # TODO: use features for each possible DA.
+cairo-lang-utils = "2"
```

Comment:
```suggestion
```
This one does not seem to be used, and it's already in the workspace (in case we need it).